improver/calibration/dataframe_utilities.py

```diff
@@ -261,6 +266,45 @@ def _training_dates_for_calibration(
     )
 
 
+def get_var_type(df: DataFrame) -> str:
+    """Check which of ALT_PERCENTILE_COLUMNS exists in the dataframe.
+
+    Args:
+        df:
+            DataFrame expected to contain exactly one of ALT_PERCENTILE_COLUMNS.
+
+    Returns:
+        var_type:
+            The member of ALT_PERCENTILE_COLUMNS found in the dataframe columns.
+
+    Raises:
+        ValueError:
+            If none of the allowed columns are present, or more than one is present.
+    """
+    var_type = None
+    for variable in ALT_PERCENTILE_COLUMNS:
+        if variable in df.columns:
+            if var_type is not None:
+                msg = (
+                    f"More than one column of {ALT_PERCENTILE_COLUMNS} "
+                    "exists in the input dataset"
+                )
+                raise ValueError(msg)
+            var_type = variable
+        else:
+            continue
+
```

Comment:
do we need to change the tests?

Reply:
You mean because of the code coverage issue? I was also confused about this, because the tests do include a case where the continue statement would be executed. Apparently this is a known issue, due to the way the python optimizer works https://github.com/nedbat/coveragepy/issues/198